understanding/adaptable.html

```diff
@@ -54,79 +52,63 @@
-                     <li><a href="#success-criteria">Success Criteria</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#success-criteria">達成基準</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				Guideline 1.3:</span>Adaptable
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				ガイドライン 1.3:</span>適応可能
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Guideline </header>
+               <header class="box-h  box-h-icon">ガイドライン</header>
                <div class="box-i">
-                  <p>Create content that can be presented in different ways (for example simpler layout)
-                     without losing information or structure.
+                  <p>情報、及び構造を損なうことなく、様々な方法 (例えば、よりシンプルなレイアウト) で提供できるようにコンテンツを制作すること。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The purpose of this guideline is to ensure that all information is available in a
-                  form that can be perceived by all users, for example, spoken aloud, or presented in
-                  a simpler visual layout. If all of the information is available in a form that can
-                  be determined by software, then it can be presented to users in different ways (visually,
-                  audibly, tactilely etc.). If information is embedded in a particular presentation
-                  in such a way that the structure and information cannot be programmatically determined
-                  by the assistive technology, then it cannot be rendered in other formats as needed
-                  by the user. 
+               <h2>意図</h2>
+               <p>このガイドラインの目的は、例えば、音声で読み上げたり、より単純なレイアウトで提示したりというように、すべての情報をすべての利用者が知覚できるように提供することである。すべての情報が、ソフトウェアによって解釈できる形式で利用可能な場合、情報を様々な方法で (視覚的、聴覚的、触覚的など) 利用者に提示できる。構造及び情報が支援技術経由でプログラムによる解釈ができないような、特定の提示で情報が埋め込まれる場合、その情報は、利用者が必要とするその他の形式でレンダリングできない。
                   
                </p>
-               <p>The Success Criteria under this guideline all seek to ensure that different types
-                  of information that are often encoded in presentation are also available so that they
-                  can be presented in other modalities. 
+               <p>このガイドラインに属する達成基準はすべて、他のモダリティで提示できるようにするために、提示で埋め込まれている様々な種類の情報も利用可能にすることを保証するよう努めるものである。
```

Comment:
2.1のものをそのまま持ってきているのですが、改めて見るとこれは結構ひどいですね。

information that are often encoded in presentation は、プレゼンテーションの中に含まれる、組み込まれるような形で表現されている情報、というようなことですね。ここでのencodeは情報をそのままではなく、何かに (ここではプレゼンテーションで表現される形に) 変えて伝えるということだと思います。言いたいこととしては「提示の中に含まれる」というくらいかと。
(もしかすると、今の訳は encode とembed を見間違えたものかもしれません……)

presentation が提示で良いのかという問題は棚上げして、素直にやると、たとえばこう?

```suggestion
               <p>このガイドラインに属する達成基準はすべて、しばしば提示の中に含まれるさまざまな種類の情報を、他のモダリティでも提示できるような形で利用可能にすることを目指している。
```

Reply:
ありがとうございます！改めて原文を読み返してみましたが、現行の日本誤訳はちょっと意味不明ですね...。 @bakera さんの修正案、よいと思います。